.github/workflows/test.yml

```diff
@@ -0,0 +1,46 @@
+name: Test Packages
+on: [push]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        node-version: [12.x]
+        os: [ubuntu-latest]
+        package: [config, expo-cli, json-file, package-manager, plist, pwa, schemer, xdl]
+    name: ${{ matrix.package }} with Node ${{ matrix.node-version }} (${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Cache node_modules
+        id: cache_node_modules
+        uses: actions/cache@master
+        with:
+          path: |
+            node_modules
+          key: ${{ runner.os }}-node-${{ hashFiles('**/yarn.lock') }}
+
+      - run: yarn
```

Comment:
```suggestion
      - run: yarn
```
It should be possible to omit this line because `lerna bootstrap` will run `yarn install`.

Reply:
But `lerna` won't be available until after yarn is run 🙃 
![qTpo0MB](https://user-images.githubusercontent.com/9664363/80020690-479e4a80-848e-11ea-9ee0-757732a06f55.gif)